.travis.yml

```diff
@@ -4,5 +4,5 @@ dist: trusty
 node_js:
 - '9.11.1'
 before_install: bash bin/ci
-script: bash bin/test
+script: yarn test
```

Comment:
I think before we can do this, we'll need to fix #1

Reply:
this preserves the same behavior as before because I have it run the tests in `yarn test`, but also runs the mocha tests (which is just one stubbed suite) and the tslint tests.

I think @jkutner was talking about converting the bash script into mocha tests though, which seems outside of this?

Reply:
see https://github.com/heroku/plugin-local-build/pull/7/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R62